libcontainer/dmz/Makefile

```diff
@@ -2,5 +2,5 @@
 include ../../cc_platform.mk
 
 runc-dmz: _dmz.c
-	$(CC) $(CFLAGS) -static -o $@ $^
+	$(CC) $(CFLAGS) -fno-asynchronous-unwind-tables -fno-ident -s -Os -nostdlib -lgcc -static -o $@ $^
```

Comment:
Could you add comments to explain these flags?
Why do we still need lgcc?

Reply:
I've added a comment to explain that, it is what it is suggested in nolibc/nolibc.h: https://github.com/opencontainers/runc/pull/4024/files#diff-bc183fb7b7c25ee341d9ef03b3bcb444301270a7ef72291bc025d96f1d7046f1R66-R68

It works without `-lgcc`, but in nolib.h they put it and the gcc manpage says that you need to add that if you use notstdlib to make sure the linking won't fail.